### PR TITLE
Added useful contract addresses

### PR DESCRIPTION
**Pull request by krofax**
Added useful contract addresses for testnet and mainnet

### CODE REVIEW

**Review by githubdoramon — CHANGES_REQUESTED** [review]
- You are mixing addresses with root hashes... Please remove the later or add as a separate section
- Why not adding etherscan/explorer links to all contracts?
- This section makes more sense under Building on zkSync,not troubleshooting

**Comment by krofax** [comment]
> https://aqwzx-zksync-v2-docs--pr332-usefull-addresses-nci4mmtu.web.app

Is there a way to link root hashes, e.g - 0xbc59c242d551e3939b9b2939b8b686efa77ba3833183045d548aa5f53357ba95

File: docs/dev/building-on-zksync/useful-address.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+| genesis_root                  | 0xbc59c242d551e3939b9b2939b8b686efa77ba3833183045d548aa5f53357ba95 |
+| genesis_tx_hash               | 0xb99ebfea46cbe05a21cd80fe5597d97b204befc52a16303f579c607dc1ac2e2e |
+
+### Token bridge contracts for mainnet
+
+:::warning
+
+_Do **not** transfer tokens or Ether to any of the addresses below; it will result in loss of funds._
+
+_Users should only interact with the token bridge contracts via [zkSync portal](https://portal.zksync.io/) or any other dapp._
```
Comment (githubdoramon) [inline docs/dev/building-on-zksync/useful-address.md:39]: zkSync Bridge (bridge.zksync.io) or zkSync Portal (portal.zksync.io)

File: docs/dev/building-on-zksync/useful-address.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+|      Contract names           | Contracts                                                                                                                          | 
+| ----------------------------- | -----------------------------------------------------------------------------------------------------------------------------------|
+| l1_erc20_bridge_proxy_addr    | [0x927DdFcc55164a59E0F33918D13a2D559bC10ce7](https://goerli.etherscan.io/address/0x927DdFcc55164a59E0F33918D13a2D559bC10ce7#code)  |
+| l1_erc20_bridge_impl_addr     | [0xE3E53270f3674965F12F70117B16736232604e12](https://goerli.etherscan.io/address/0xE3E53270f3674965F12F70117B16736232604e12#code)  |
+| l2_erc20_bridge_addr          | [0x00ff932A6d70E2B8f1Eb4919e1e09C1923E7e57b](https://goerli.explorer.zksync.io/address/0x00ff932A6d70E2B8f1Eb4919e1e09C1923E7e57b) |
+
+
+
+
+## Token request
```
Comment (githubdoramon) [inline docs/dev/building-on-zksync/useful-address.md:91]: Known tokens

File: docs/dev/building-on-zksync/useful-address.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
+# Useful Addresses
+
+Here are some contract addresses that may be useful and/or of interest to those exploring the zkSync Era:
+
+## Mainnet contract addresses
+
+|      Contract names           | zkSync Era Mainnet                                                 |
```
Comment (githubdoramon) [inline docs/dev/building-on-zksync/useful-address.md:7]: Contract Name | Address

File: docs/dev/building-on-zksync/useful-address.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+| governance_facet_addr         | [0xa7E8a8F71c3cC43946601CC99997f8Cd6828a9B9](https://etherscan.io/address/0xa7E8a8F71c3cC43946601CC99997f8Cd6828a9B9#code)|
+| verifier_addr                 | [0x6A28B1bC6123fE525FF46d881Ff80871D0Eb3F02](https://etherscan.io/address/0x6A28B1bC6123fE525FF46d881Ff80871D0Eb3F02#code)|
+| mailbox_facet_addr            | [0x2EA0CFB9C942058ee5A84411EF2E37C6DE5bfe5c](https://etherscan.io/address/0x2EA0CFB9C942058ee5A84411EF2E37C6DE5bfe5c#code)|
+| l1_allow_list_addr            | [0x185e050EC1c22c45445a9AdEA99B12e1A1372eBA](https://etherscan.io/address/0x185e050EC1c22c45445a9AdEA99B12e1A1372eBA#code)|
+| l2_testnet_paymaster_addr     | [0x0265d9a5af8af5fe070933e5e549d8fef08e09f4](https://etherscan.io/address/0x0265d9a5af8af5fe070933e5e549d8fef08e09f4#code)|
+
+
+## Root and hashes
+
+|       Contract names          | zkSync Era Mainnet                                                 |
```
Comment (githubdoramon) [inline docs/dev/building-on-zksync/useful-address.md:24]: Name | Hash

File: docs/dev/building-on-zksync/useful-address.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+
+:::warning
+
+_Do **not** transfer tokens or Ether to any of the addresses below; it will result in loss of funds._
+
+_Users should only interact with the token bridge contracts via [zkSync portal](https://portal.zksync.io/) or any other dapp._
+
+:::
+
+|      Contract names           | Contracts                                                                                                                 |
```
Comment (githubdoramon) [inline docs/dev/building-on-zksync/useful-address.md:43]: Contract Name | Address

File: docs/dev/building-on-zksync/useful-address.md [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+| l1_erc20_bridge_proxy_addr    | [0x57891966931Eb4Bb6FB81430E6cE0A03AAbDe063](https://etherscan.io/address/0x57891966931Eb4Bb6FB81430E6cE0A03AAbDe063#code)|
+| l1_erc20_bridge_impl_addr     | [0x22bE40122BA952b81c50bcaa86F84418ff623391](https://etherscan.io/address/0x22bE40122BA952b81c50bcaa86F84418ff623391#code)|
+| l2_erc20_bridge_addr          | [0x11f943b2c77b743AB90f4A0Ae7d5A4e7FCA3E102](https://etherscan.io/address/0x11f943b2c77b743AB90f4A0Ae7d5A4e7FCA3E102#code)|
+
+
+
+## Testnet contract addresses
+
+
+|      Contract names           | L1                                                                                                                               |
```
Comment (githubdoramon) [inline docs/dev/building-on-zksync/useful-address.md:54]: Contract Name | Address

File: docs/dev/building-on-zksync/useful-address.md [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+| governance_facet_addr         | [0xc288177781D3555822edB31D323aEcB6cFD849c7](https://goerli.etherscan.io/address/0xc288177781D3555822edB31D323aEcB6cFD849c7#code)|
+| verifier_addr                 | [0xc8517230276e0df51377ecc07b528cd3ee083132](https://goerli.etherscan.io/address/0xc8517230276e0df51377ecc07b528cd3ee083132#code)|
+| mailbox_facet_addr            | [0xd80eF7aCBEC07dbf10Eb84452b40D0a8882ADfB5](https://goerli.etherscan.io/address/0xd80eF7aCBEC07dbf10Eb84452b40D0a8882ADfB5#code)|
+| l1_allow_list_addr            | [0xCbA757b4f0527b535bE80720325064058FC4A306](https://goerli.etherscan.io/address/0xCbA757b4f0527b535bE80720325064058FC4A306#code)|
+| l2_testnet_paymaster_addr     | [0x8f0ea1312da29f17eabeb2f484fd3c112cccdd63](https://goerli.explorer.zksync.io/address/0x8f0ea1312da29f17eabeb2f484fd3c112cccdd63#contract)|
+
+
+## Root and hashes
+
+|                               | zkSync Era Testnet                                                 |
```
Comment (githubdoramon) [inline docs/dev/building-on-zksync/useful-address.md:71]: Name | Hash

File: docs/dev/building-on-zksync/useful-address.md [hunk 8]
```diff
@@ -0,0 +1,98 @@
...
+| vk_commitment_basic_circuits  | 0x142a364ef2073132eaf07aa7f3d8495065be5b92a2dc14fda09b4216affed9c0 |
+| vk_commitment_leaf            | 0x101e08b00193e529145ee09823378ef51a3bc8966504064f1f6ba3f1ba863210 |
+| vk_commitment_node            | 0x1186ec268d49f1905f8d9c1e9d39fc33e98c74f91d91a21b8f7ef78bd09a8db8 |
+| genesis_block_commitment      | 0x60d946bb425a7a5f6104fe7ef4b854fac4c32b6ba5b3601c82014fa28f6511d6 |
+| genesis_root                  | 0xc4b302452a77d72c6ad618ab1f18ec3a7699520dc053e767eca6427d24bd0848 |
+| genesis_tx_hash               | 0xb99ebfea46cbe05a21cd80fe5597d97b204befc52a16303f579c607dc1ac2e2e |
+
+### Token bridge contracts for testnet
+
+|      Contract names           | Contracts                                                                                                                          |
```
Comment (githubdoramon) [inline docs/dev/building-on-zksync/useful-address.md:82]: Contract Name | Address

File: docs/dev/building-on-zksync/useful-address.md [hunk 9]
```diff
@@ -0,0 +1,98 @@
...
+| executor_facet_addr           | [0x98E900Eb2e5fdE9786f736e86d6BFBfDb3E4683b](https://etherscan.io/address/0x98E900Eb2e5fdE9786f736e86d6BFBfDb3E4683b#code)|
+| getters_facet_addr            | [0xc30af84A6aFF43D5fd06FEdA6bA20BdfeD539F6C](https://etherscan.io/address/0xc30af84A6aFF43D5fd06FEdA6bA20BdfeD539F6C#code)|
+| governance_facet_addr         | [0xa7E8a8F71c3cC43946601CC99997f8Cd6828a9B9](https://etherscan.io/address/0xa7E8a8F71c3cC43946601CC99997f8Cd6828a9B9#code)|
+| verifier_addr                 | [0x6A28B1bC6123fE525FF46d881Ff80871D0Eb3F02](https://etherscan.io/address/0x6A28B1bC6123fE525FF46d881Ff80871D0Eb3F02#code)|
+| mailbox_facet_addr            | [0x2EA0CFB9C942058ee5A84411EF2E37C6DE5bfe5c](https://etherscan.io/address/0x2EA0CFB9C942058ee5A84411EF2E37C6DE5bfe5c#code)|
+| l1_allow_list_addr            | [0x185e050EC1c22c45445a9AdEA99B12e1A1372eBA](https://etherscan.io/address/0x185e050EC1c22c45445a9AdEA99B12e1A1372eBA#code)|
+| l2_testnet_paymaster_addr     | [0x0265d9a5af8af5fe070933e5e549d8fef08e09f4](https://etherscan.io/address/0x0265d9a5af8af5fe070933e5e549d8fef08e09f4#code)|
```
Comment (githubdoramon) [inline docs/dev/building-on-zksync/useful-address.md:19]: how can the testnet paymaster be on mainnet addresses?

**Review by githubdoramon — CHANGES_REQUESTED** [review]
Needs some changes... and you linked L2 contracts to etherscan instead of our explorer

File: docs/dev/building-on-zksync/useful-address.md [hunk 10]
```diff
@@ -16,12 +16,12 @@ Here are some contract addresses that may be useful and/or of interest to those
 | verifier_addr                 | [0x6A28B1bC6123fE525FF46d881Ff80871D0Eb3F02](https://etherscan.io/address/0x6A28B1bC6123fE525FF46d881Ff80871D0Eb3F02#code)|
 | mailbox_facet_addr            | [0x2EA0CFB9C942058ee5A84411EF2E37C6DE5bfe5c](https://etherscan.io/address/0x2EA0CFB9C942058ee5A84411EF2E37C6DE5bfe5c#code)|
 | l1_allow_list_addr            | [0x185e050EC1c22c45445a9AdEA99B12e1A1372eBA](https://etherscan.io/address/0x185e050EC1c22c45445a9AdEA99B12e1A1372eBA#code)|
-| l2_testnet_paymaster_addr     | [0x0265d9a5af8af5fe070933e5e549d8fef08e09f4](https://etherscan.io/address/0x0265d9a5af8af5fe070933e5e549d8fef08e09f4#code)|
+| l2_mainnet_paymaster_addr     | [0x0265d9a5af8af5fe070933e5e549d8fef08e09f4](https://goerli.explorer.zksync.io/address/0x0265d9a5af8af5fe070933e5e549d8fef08e09f4)|
```
Comment (githubdoramon) [inline docs/dev/building-on-zksync/useful-address.md:19]: @krofax there is no mainnet paymaster, and especially not pointing to a goerli address. Just remove this line

File: docs/dev/building-on-zksync/useful-address.md [hunk 11]
```diff
@@ -36,22 +36,23 @@ Here are some contract addresses that may be useful and/or of interest to those
...
-| l2_erc20_bridge_addr          | [0x11f943b2c77b743AB90f4A0Ae7d5A4e7FCA3E102](https://etherscan.io/address/0x11f943b2c77b743AB90f4A0Ae7d5A4e7FCA3E102#code)|
+| l2_erc20_bridge_addr          | [0x11f943b2c77b743AB90f4A0Ae7d5A4e7FCA3E102](https://explorer.zksync.io/address/0x11f943b2c77b743AB90f4A0Ae7d5A4e7FCA3E102)|
 
 
 
 ## Testnet contract addresses
 
 
-|      Contract names           | L1                                                                                                                               | 
+|      Contract Name            | L1 Address                                                                                                                       |
```
Comment (githubdoramon) [inline docs/dev/building-on-zksync/useful-address.md:55]: Remove the L1 and make consitent to others

File: docs/dev/building-on-zksync/useful-address.md [hunk 12]
```diff
@@ -63,12 +64,12 @@ _Users should only interact with the token bridge contracts via [zkSync portal](
 | verifier_addr                 | [0xc8517230276e0df51377ecc07b528cd3ee083132](https://goerli.etherscan.io/address/0xc8517230276e0df51377ecc07b528cd3ee083132#code)|
 | mailbox_facet_addr            | [0xd80eF7aCBEC07dbf10Eb84452b40D0a8882ADfB5](https://goerli.etherscan.io/address/0xd80eF7aCBEC07dbf10Eb84452b40D0a8882ADfB5#code)|
 | l1_allow_list_addr            | [0xCbA757b4f0527b535bE80720325064058FC4A306](https://goerli.etherscan.io/address/0xCbA757b4f0527b535bE80720325064058FC4A306#code)|
-| l2_testnet_paymaster_addr     | [0x8f0ea1312da29f17eabeb2f484fd3c112cccdd63](https://goerli.explorer.zksync.io/address/0x8f0ea1312da29f17eabeb2f484fd3c112cccdd63#contract)|
+| l2_testnet_paymaster_addr     | [0x8f0ea1312da29f17eabeb2f484fd3c112cccdd63](https://goerli.explorer.zksync.io/address/0x8f0ea1312da29f17eabeb2f484fd3c112cccdd63)|
```
Comment (githubdoramon) [inline docs/dev/building-on-zksync/useful-address.md:67]: Why removing the #contract?

**Review by githubdoramon — CHANGES_REQUESTED** [review]

File: docs/dev/building-on-zksync/useful-address.md [hunk 13]
```diff
@@ -0,0 +1,99 @@
...
+|      Contract Name           | Address                                                                                                                          | 
+| ----------------------------- | -----------------------------------------------------------------------------------------------------------------------------------|
+| l1_erc20_bridge_proxy_addr    | [0x927DdFcc55164a59E0F33918D13a2D559bC10ce7](https://goerli.etherscan.io/address/0x927DdFcc55164a59E0F33918D13a2D559bC10ce7#code)  |
+| l1_erc20_bridge_impl_addr     | [0xE3E53270f3674965F12F70117B16736232604e12](https://goerli.etherscan.io/address/0xE3E53270f3674965F12F70117B16736232604e12#code)  |
+| l2_erc20_bridge_addr          | [0x00ff932A6d70E2B8f1Eb4919e1e09C1923E7e57b](https://goerli.explorer.zksync.io/address/0x00ff932A6d70E2B8f1Eb4919e1e09C1923E7e57b) |
+
+
+
+
+## Known tokens
```
Comment (githubdoramon) [inline docs/dev/building-on-zksync/useful-address.md:92]: @krofax pls add token name and symbol (same ones available at Etherescan on L1)

**Pull request merged**